services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/api_client/_public.py

```diff
@@ -399,6 +401,19 @@ async def restart_containers(self, dynamic_sidecar_endpoint: AnyHttpUrl) -> None
             _debug_progress_callback,
         )
 
+    @log_decorator(logger=logger)
```

Comment:
maybe go a bit down with the logs. they tend to clutter a lot.

Reply:
I would do that in a separate PR. Not here.